core/templates/dev/head/pages/about/about.html

```diff
@@ -508,6 +508,9 @@
     .oppia-about-foundation-tab-anchor:hover {
       text-decoration: none;
     }
+    .oppia-about-text-align {
```

Comment:
@seanlip -- is it okay that this class is declared in this file, or should it go into the main CSS file?